editor/src/core/data-tracing/data-tracing.ts

```diff
@@ -279,63 +279,67 @@ function processJSPropertyAccessorsIntoDataTracingPath(
 }
 
 function propUsedByIdentifierOrAccess(
-  param: Param,
+  params: Array<Param>,
   originalIdentifier: JSIdentifier,
   pathDrillSoFar: DataPathPositiveResult,
 ): Either<string, { propertyName: string; modifiedPathDrillSoFar: DataPathPositiveResult }> {
-  function getPropertyNameFromPathSoFar(): Either<
-    string,
-    { propertyName: string; modifiedPathDrillSoFar: DataPathPositiveResult }
-  > {
-    switch (pathDrillSoFar.type) {
-      case 'DATA_PATH_SUCCESS':
-        const propertyName = pathDrillSoFar.dataPath.at(0)
-        if (propertyName == null) {
-          return left('Path so far is empty.')
-        } else {
-          return right({
-            propertyName: propertyName,
-            modifiedPathDrillSoFar: dataPathSuccess(pathDrillSoFar.dataPath.slice(1)),
-          })
-        }
-      case 'DATA_PATH_NOT_POSSIBLE':
-        return left('Path is not available.')
-      default:
-        assertNever(pathDrillSoFar)
-    }
-  }
-  switch (param.boundParam.type) {
-    case 'REGULAR_PARAM': {
-      // in case of a regular prop param, first we want to match the param name to the original identifier
-      if (param.boundParam.paramName !== originalIdentifier.name) {
-        return left('identifier does not match the prop name')
+  for (const param of params) {
```

Comment:
hm I think this code now erroneously assumes that every parameter is a `props` param. we should instead assume that the first parameter is `props`

Reply:
Let me have a looky see...

Reply:
Fixed this in 1645db31b.